i18n/GLOSSARY_fr.md

```diff
@@ -19,6 +19,13 @@ Ce glossaire accompagne les [principes GitOps](./PRINCIPLES_fr.md), et autres do
 
     Lorsque l'état réel d'un système s'est éloigné ou est en train de s'éloigner de [l'état souhaité](#etat-souhaite), on parle alors de dérive.
 
+- ## Pull
```

Comment:
I had it translated as  **Extraction ("Pull")** but not quite sure what people use in native French speaking areas.

Reply:
If I modify it to 'Extraction' ('Pull'), won't it change the header anchor?

Reply:
> If I modify it to 'Extraction' ('Pull'), won't it change the header anchor?

Yes, and that's OK, as long as we check that the anchors in any links to those are also updated to match 😄

Reply:
PS thanks @juliafmorgado for pointing out that neither of the 2 links from the Glossary to principle 3 and principle 4 point to their anchors, but only to the page itself. We can fix that in the English version too.

Reply:
I ended up changing it to **Extraction ("Pull")**